security/index.html

```diff
@@ -2,7 +2,16 @@
 layout: default
 title: "Ruby on Rails: Security Policy"
 ---
-    <div id="article" class="security"><article>
+    <div id="article"><article>
+      <header>
+        <h1>Security policy</h1>
+        <h2>
+          Did you find a security vulnerability in Ruby on Rails?<br />
+          Follow these steps to safely report the issue to the core team.<br />
+          You might even be eligible for a bounty.
```

Comment:
Would you only remove it from the header. Or would you actually change [this line of the page](https://github.com/rails/rails.github.com/blob/master/security/index.html#L63) where it says:

> All security bugs in rails should be reported through our <a href="https://hackerone.com/rails">bounty program page at HackerOne</a>.